src/installer/corehost/cli/apphost/static/CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+# Licensed to the .NET Foundation under one or more agreements.
+# The .NET Foundation licenses this file to you under the MIT license.
+# See the LICENSE file in the project root for more information.
+
+project(static_apphost)
```

Comment:
@richlander  @samsp-msft  
Wanted to ask again if we want a better name for the static host, instead of static_apphost.exe? 

We build two types of hosts:
* Apphost linked with other host components code-named "static apphost" (this PR) -- this will ship in the Microsoft.netcoreapp.host package for Windows (and Mac).
* Apphost linked with host and runtime components code-named "super apphost" (TBD) -- this will ship in the Microsoft.netcoreapp.host package for Linux.

Reply:
@VSadov  please see @jkotas  comments here: https://github.com/dotnet/runtime/issues/32823#issuecomment-598458767.
This `static_apphost` should be renamed to `netcoreapphost` or `singlefilehost`

Reply:
The file is a template to be linked with app parts and renamed, so naming is not extremely important IMO. 
`singleapphost` feels a bit better as it hints at the purpose of the file.
I assume the standalone apphost will not be used (or will not be recommended) for single-file scenarios once this is shipped.

I will change to `singleapphost`.

Reply:
The feature is called SingleFile (e.g. `-p:PublishSingleFile=true`) so the name should be `singlefile*` for consistency.

Reply:
`singlefilehost` ?

Reply:
Actually, I meant `singlefilehost` in the first place. 

`singleapphost` was a mistype.